specificatie/geheimhouding.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.0.3
+info:
+  title: Geheimhouding definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    AbstractGeheimhoudingPersoonsgegevens:
```

Comment:
is polyformisme hier nog wel de makkelijkste oplossing? Dus makkelijker dan gewoon de LO GBA waarde leveren?

Ook na de proxy moet een gebruiker in deze oplossing eerst een conditie op geheimhoudingPersoonsgegevens.type hebben voordat geheimhoudingPersoonsgegevens.geheimhouding=true wordt afgetest
Je verlies hiermee helemaal het gemak van simpel uitlezen en aftesten op `if geheimhoudingPersoonsgegevens==true then`

Dan kunnen we denk ik net zo goed hier direct een waardetabel van maken (geen informatievraag) met code en omschrijving.
Dan kan een gebruiker hier `if geheimhoudingPersoonsgegevens is not nul then` op doen, en maakt het niet meer uit wat erin staat, zolang we de leegwaarde (0) er maar uitfilteren

Reply:
wat mij betreft een goede design keuze. Ik zal dit doorvoeren als er geen bezwaren zijn.

Reply:
nadeel van deze keuze is wel dat we hier niet meer zo makkelijk vanaf komen. Zou het een oplossing zijn dat de GBA versie (RvIG versie) geheimhouding levert in een tijdelijk (direct depricated) nieuw property "indicatieGeheim" in integer vorm en de proxy die naar boolean vertaalt in geheimhoudingPersoonsgegevens?
is niet zo'n charmante (maar complexere) oplossing als polymorfisme, maar de gebruiker achter de proxy merkt daar niks van (krijgt gewoon een simpele boolean)

Reply:
Wat mij betreft prima, maar dan is het beter om het consistent door te trekken naar alle andere informatie types en daar ook een depricated gba property introduceren, dus bijv. een depricated gbadatum property bij onvolledigdatum en indien nodig indicatieOnbekend property. Dan is het niet nodig om polymorfisme toe te passen